src/components/Player/Pages/Heroes/playerHeroesColumns.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router';
 import { transformations, getPercentWin } from 'utility';
 import { TablePercent } from 'components/Visualizations';
 import strings from 'lang';
 
-export const playerHeroesOverviewColumns = [{
+export const playerHeroesOverviewColumns = playerId => [{
   displayName: strings.th_hero_id,
   tooltip: strings.tooltip_hero_id,
   field: 'hero_id',
-  displayFn: transformations.hero_id,
+  displayFn: row => <Link to={`/players/${playerId}/matches?hero_id=${row.hero_id}`}>{transformations.hero_id(row)}</Link>,
```

Comment:
the filter link for peers/pros is on the column with the number of games.  For heroes it should be the same (so the column with field: 'games'?